operator/src/main/resources/instances/managedkafka.properties

```diff
@@ -31,3 +31,15 @@ storage.formatting-overhead=0.05
 
 # Override for partition limit enforcement. This override should be removed once we can enable this by default
 kafka.partition-limit-enforced=false
+
+cruisecontrol.metric-reporter-topic=__redhat_strimzi.cruisecontrol.metrics
```

Comment:
I remember in another issue where we are renaming from "_redhat" to "_kas"?

Reply:
@rareddy are you able to find that issue? I've had a look but can't find it, and don't want to rename unless we've changed the internal logic that determines which topics should be hidden from customers

Reply:
I think __redhat is okay here for now.  I think there is a separate task to de-Red Hat kas-fleetshard as part of its open-source story.  We can tackle all the __redhat references in one go.

Reply:
sounds good (y)